tst/EnergyPlus/unit/InputProcessor.unit.cc

```diff
@@ -2890,19 +2890,20 @@ TEST_F(InputProcessorFixture, getObjectItem_unitary_system_input)
                                                              "",
                                                              "",
                                                              "",
+                                                             "",
                                                              ""}),
                                    Alphas));
     EXPECT_TRUE(
         compare_containers(std::vector<bool>({false, false, false, false, false, false, false, false, false, false, false, false, false, false,
-                                              false, true,  true,  false, false, false, false, false, true,  true,  true,  true,  true}),
+                                              false, true,  true,  false, false, false, false, false, true,  true,  true,  true,  true, true}),
                            lAlphaBlanks));
 
     EXPECT_EQ(17, NumNumbers);
     EXPECT_TRUE(compare_containers(std::vector<bool>({true, true, false, true,  true, true, false, true, true, true, false, true, true,
-                                                      true, true, true,  false, true, true, true,  true, true, true, true,  true, true}),
+                                                      true, true, true,  false, true, true, true,  true, true, true, true,  true, true, true}),
                                    lNumericBlanks));
     EXPECT_TRUE(
-        compare_containers(std::vector<Real64>({1, 2, 1.6, 0, 0, 0, 1.6, 0, 0, 0, 1.6, 0, 0, 0, 0, 0, 80, 0, 0, 0, 0, 0, 0, 0, 0, 0}), Numbers));
+        compare_containers(std::vector<Real64>({1, 2, 1.6, 0, 0, 0, 1.6, 0, 0, 0, 1.6, 0, 0, 0, 0, 0, 80, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0}), Numbers));
     EXPECT_EQ(1, IOStatus);
```

Comment:
These changes were made to correct a failed unit test due to the two new input fields added in UnitarySystem. Otherwise, the unit test changes have to reverted when the two input fields are moved to the new object, CoilSystem:Cooling:Water.